### ISSUE TITLE
Cannot use GitHub token: `Cannot deserialize GitHub API content: Error("invalid type: map, expected a sequence", line: 1, column: 0)', src/github_events/mod.rs:52:38`

### ISSUE THREAD

**Issue by nicokosi** [nicokosi/pullpito#98]
```sh
$ git show | head -1
commit 06e46c53c22e49253ba501dab42a9e7ef755ac93

$ cargo run -- --repository softwarevidal/nemo --token $GITHUB_TOKEN
    Finished dev [unoptimized + debuginfo] target(s) in 0.14s
     Running `target/debug/pullpito --repository softwarevidal/nemo --token ***`
thread '<unnamed>' panicked at 'Cannot deserialize GitHub API content: Error("invalid type: map, expected a sequence", line: 1, column: 0)', src/github_events/mod.rs:52:38
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I am able to query https://api.github.com/repos/softwarevidal/nemo/events?page=1 with the same GitHub token.

**Comment by nicokosi** [comment]
As seen via Postman, response header is:
`Link: <https://api.github.com/repositories/257951013/events?page=2>; rel="next", <https://api.github.com/repositories/257951013/events?page=10>; rel="last"`

**Comment by nicokosi** [comment]
When debugging, I can see that GitHub API's response is: 😇 
```
{"message":"Must specify access token via Authorization header. https://developer.github.com/changes/2020-02-10-deprecating-auth-through-query-param","documentation_url":"https://docs.github.com/v3/#oauth2-token-sent-in-a-header"}
```

**Comment by nicokosi** [comment]
Fixed in commit 430403449d7fd2ee72a2ad6ad4ee678f7e6c26d5 in [tag v1.0.1](https://github.com/nicokosi/pullpito/releases/tag/v1.0.1).
